tools/transifex/.tx/config

```diff
@@ -89,3 +89,41 @@ trans.pt_BR = work-dir/messages_pt_BR.properties
 trans.ru_RU = work-dir/messages_ru_RU.properties
 trans.zh_CN = work-dir/messages_zh_CN.properties
 
+[CloudStack_UI.48xmessagesproperties]
```

Comment:
Shouldn't this change be targeted against the 4.8 branch?  Seems strange that we would be merging it into 4.9 and it applies to 4.8 but is not in the 4.8 source code.  Thoughts?